shared/editor/components/Styles.ts

```diff
@@ -733,7 +733,7 @@ img.ProseMirror-separator {
   }
 }
 
-.heading-name {
+.scroll-to-anchor-heading, .scroll-to-anchor-image {
```

Comment:
Not really sure why the rename to include `scroll-to-`. Probably best to just move the class name into `EditorStyleHelper` as a constant as it's used across files 👍